tequilapi/endpoints/service.go

```diff
@@ -106,13 +115,15 @@ func (se *ServiceEndpoint) ServiceList(resp http.ResponseWriter, _ *http.Request
 //     schema:
 //       "$ref": "#/definitions/ErrorMessageDTO"
 func (se *ServiceEndpoint) ServiceGet(resp http.ResponseWriter, _ *http.Request, params httprouter.Params) {
-	statusResponse := toServiceInfoResponse(se.serviceState)
-	statusResponse.ID = params.ByName("id")
-	if statusResponse.ID != "6ba7b810-9dad-11d1-80b4-00c04fd430c8" && statusResponse.ID != "7037d6f6-1b92-4513-8cb4-8fd1a944b996" {
-		utils.SendErrorMessage(resp, "Service not found", http.StatusNotFound)
-		return
+	for _, instance := range se.serviceManager.List() {
```

Comment:
this lookup could also be made easier, if we'd have a map instead of a list there :)

Reply:
Changed it.